lib/src/widget/category_card.dart

```diff
@@ -48,13 +48,6 @@ class CategoryCard extends StatelessWidget {
                     style: theme.textTheme.titleMedium,
                   ),
                 ),
-                Padding(
```

Comment:
I don't consider this a fix and it looks more like a workaround.

Maybe you @Teifun2 can tell me how you got the error (screen size, resolution, system text scale, global system scaling ...)

A screenshot is always useful. I think we can find a better real fix.
Also you should probably run `dart analyze` before commiting or look in your IDE for warnings as you forgot to remove the variable.

As say